rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -317,6 +317,13 @@ public String deployNodeSource(final String sessionId, final String nodeSourceNa
                                                                                              nodeSourceName));
     }
 
+    @Override
+    public String undeployNodeSource(final String sessionId, final String nodeSourceName)
+            throws RestServerException, ServiceException {
+        return executeFunctionReturnStreamAsString(restClient -> restClient.undeployNodeSource(sessionId,
+                                                                                               nodeSourceName));
+    }
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)